packages/telemetry/telemetry.ts

```diff
@@ -246,3 +251,64 @@ export default class Telemetry {
     }
   }
 }
+
+class GitProperties {
+  static async contributors(sinceDaysAgo: number, cwd?: PathLike): Promise<Array<string>> {
+    const unixTimeNow = Math.floor(Number(new Date()) / 1000);
+    const unixTimeAgo = unixTimeNow - sinceDaysAgo * 24 * 60 * 60;
+    try {
+      const { stdout } = await exec(
+        [
+          'git',
+          cwd && `-C ${cwd.toString()}`,
+          '--no-pager',
+          'log',
+          `--since=${unixTimeAgo}`,
+          '--format="%ae"',
+        ].join(' ')
+      );
+      return [
+        ...stdout
+          .trim()
+          .split('\n')
+          .reduce((acc, email) => {
+            acc.add(email);
+            return acc;
+          }, new Set<string>()),
+      ];
+    } catch {
+      return [];
```

Comment:
Keep in mind, if contributor count is `[]` it means that either git is not available, or that the project has no git repo, or that there are no commits. Should we indicate those situations as specific telemetry conditions so we don't have to guess? It would be nice to be able to run a report like what % of users have git on their system, what % of projects are in a git repo.